nss/src/logs/mod.rs

```diff
@@ -15,7 +15,7 @@ macro_rules! info {
 macro_rules! error {
```

Comment:
Maybe change the `macro_rule!` to info?

Reply:
there is one just above for it, no? I’m even unsure this is executed TBH because I didn’t see the prefix and the error was using info(), so probably something to clean up? (which is outside of this PR I guess)?

Reply:
Well, it's the macro that's being exported when you `use crate::{error}`. If you are switching to `log::info!` (which, yeah, is the same as above) we can remove this macro completely (if we are not using it elsewhere).

Reply:
Thanks for clearing that up!